etc/proto/run.sh

```diff
@@ -41,7 +48,19 @@ Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,\
     "${i}" >/dev/stderr
 done
 
-# TODO (brendon): figure out how to configure protoc
-cd v2
+pushd src > /dev/stderr
+read -ra proto_files < <(find . -name "*.proto" -print0 | xargs -0)
+protoc \
+    --proto_path . \
+    --plugin=protoc-gen-pach="${GOPATH}/bin/protoc-gen-pach" \
+    "-I${GOPATH}/pkg/mod/github.com/gogo/protobuf@${GOGO_PROTO_VERSION}" \
+    --pach_out="../v2/src" \
+    "${proto_files[@]}" > /dev/stderr
 
+popd > /dev/stderr
+
+
+# TODO (brendon): figure out how to configure protoc
+pushd v2 > /dev/stderr
+gofmt -w . > /dev/stderr
```

Comment:
The original form of the generated code had some formatting issues with indentation and import sort order. The indentation issue is easily solved, but I couldn't figure out how to sort a `range` in Go templates.

Reply:
Hmm I tried `goimports`, but it also changes other generated protos.